css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "DM sans", sans-serif;
+  color: #e6e6e6;
+}
+
+.container {
+  width: 1055px;
+  margin: 0 auto;
+}
+
+.heading {
+  text-align: center;
+  color: black;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  width: 20%;
+  margin: 0 1%;
+}
```

Comment:
This `column` class still needs an appropriate name that identifies its size. Also, the `margin` should be applied to a separate class that you can add in addition to this class. `column` classes should only include `flex-basis` or `width`, nothing else.